.gitignore

```diff
@@ -7,4 +7,5 @@ Frontend/public/*
 *.sqlite
 data
 *.log
-.*.txt
\ No newline at end of file
+.*.txt
```

Comment:
You can remove *.txt it was a hack to keep any swarm secrets out of git.

Reply:
cool will do